django_cloud_tasks/tasks.py

```diff
@@ -221,6 +222,10 @@ class PublisherTask(Task):
     # might be useful to use the Cloud Task throttling
     publish_immediately = True
 
+    def __init__(self, enable_message_ordering=False, **kwargs) -> None:
+        super().__init__()
+        self.enable_message_ordering = enable_message_ordering
```

Comment:
Add the attribute explicitly to the init, otherwise we'll have to figure out which attributes can be used.